addon-test-support/ember-cli-qunit.js

```diff
@@ -51,14 +51,20 @@ export function setupTestContainer() {
   let params = QUnit.urlParams;
 
   let containerVisibility = params.nocontainer ? 'hidden' : 'visible';
-  let containerPosition = (params.dockcontainer || params.devmode) ? 'absolute' : 'relative';
+  let containerPosition = (params.dockcontainer || params.devmode) ? 'fixed' : 'relative';
 
   if (params.devmode) {
     testContainer.className = ' full-screen';
   }
 
   testContainer.style.visibility = containerVisibility;
   testContainer.style.position = containerPosition;
+
+  let qunitContainer = document.getElementById('qunit');
+  let testContainerHeight = window.getComputedStyle(testContainer).height;
+  if (qunitContainer && params.dockcontainer && testContainerHeight) {
```

Comment:
I think we can simplify this. The `#qunit` div is inserted at build time so it should always be there and I don't think we care if `testContainerHeight` is truthy or not, since setting it below won't be affected (as far as I know).

Basically, I think we can just do:

```js
if (params.dockcontainer) {
  qunitContainer.style.marginBottom = window.getComputedStyle(testContainer).height;
}
```